sound/soc/sof/ipc4-topology.c

```diff
@@ -1494,6 +1494,53 @@ sof_ipc4_prepare_copier_module(struct snd_sof_widget *swidget,
 	u32 deep_buffer_dma_ms = 0;
 	int output_fmt_index;
 
+	/* just modify the input params for the next widget if the widget is already prepared */
```

Comment:
commit message does not parse for me

In preparation for handling repeated ipc_prepare calls for the same
widget, add the logic to simple update the pipeline params if the
copier is already prepared.

did you mean 'simply update', or maybe 'perform a simple update'

confusion between update as a verb or a noun